spec/config/ruby.yml

```diff
@@ -1,2 +1,2 @@
-gemName: conjur
-moduleName: Conjur
+gemName: conjur-sdk
+moduleName: ConjurSDK
```

Comment:
Is there a reason why the config here is different from the Python config naming scheme? Might be nice for them to be as uniform as possible across clients

Reply:
The reason I chose this naming scheme is because the new generated client repositories will be named `conjur-sdk-<language>` so this scheme fits in with that and differentiates the generated client from the current one. The Python client should end up named something similar to fit in with the others but it is outside the scope of this PR.